third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -447,10 +655,20 @@ JITModuleDescriptor* JITCompiler::compile() {
         emitBinary(compiler_, item->asInstruction());
         break;
       }
+      case Instruction::BinaryFloat32:
+      case Instruction::BinaryFloat64: {
```

Comment:
It seems you use the same group. Is it worth to have two groups for them? Same is true for unary ops.